### DISCUSSION TITLE
Issue with Displaying Translations in Tabular Editor When Switching Languages

### DISCUSSION THREAD

**Discussion by SokhnaDieng** [TabularEditor/TabularEditor3#1271] [Data Modeling]
Hello,

I am using Tabular Editor to handle the translation of column names in our data model for these languages, which works fine, an issue arises when I switch to a different interface language, such as Spanish (ES).
![image](https://github.com/TabularEditor/TabularEditor3/assets/165020579/04300d3c-847a-45b0-8f3a-83b30c89019b)
![image](https://github.com/TabularEditor/TabularEditor3/assets/165020579/e8167349-f30c-4b75-a792-eac0cc78a280)
![image](https://github.com/TabularEditor/TabularEditor3/assets/165020579/72785a9e-9c34-4d31-8173-17cc6388ad61)
![image](https://github.com/TabularEditor/TabularEditor3/assets/165020579/bcd9267f-b7e7-45b5-b861-cc514ea23c7f)
Here's the problem: Although the user interface is set to Spanish, measures still display in English, which is the default translation language in our model, and other columns show their technical names instead of the default language. This makes the model difficult to read and understand for our Spanish-speaking users for example.

![image](https://github.com/TabularEditor/TabularEditor3/assets/165020579/cebe4cbf-b8db-4e34-8ecb-b131a3e32aa5)

Are there specific settings or steps that I should be following to ensure that when the interface is set to another language, all translations, including measures, are displayed correctly in the default language? Is there a configuration that I might have missed?

Any help or pointers towards relevant documentation would be greatly appreciated.

Thank you in advance for your support.

**Comment by otykier** [comment]
Metadata translations should work with Power BI Premium in the web browser. I am not sure if it works in Power BI Desktop, though. Please read through [this blog post](https://www.kasperonbi.com/setting-up-translations-for-power-bi-premium/) by Kasper de Jonge from the Fabric CAT team, to learn more, or reach out to Microsoft.
